source/Lib/DecoderLib/BinDecoder.cpp

```diff
@@ -129,40 +133,46 @@ unsigned BinDecoder::decodeBinsEP( unsigned numBins )
   }
   unsigned remBins = numBins;
   unsigned bins    = 0;
+  int value = m_Value;
+  int range = m_Range;
+  int bitsNeeded = m_bitsNeeded;
```

Comment:
Does this impact? I remember trying this, without success. I think the compiler can take care of localizing the memory in this function, no?

Reply:
After optimization, using Xcode's profile tool, the time of decodeBinsEP is reduced.

Some compilers can perform this optimization by themselves, but the optimized code is more friendly to the compiler.